src/components/dashboard/WhoContent.native.js

```diff
@@ -25,6 +28,17 @@ const WhoContent = ({ styles, setValue, error, text, value, next, state }) => {
     })
   }
 
+  const selectContact = (name, phone) => {
+    setName(name)
+    setPhone(phone)
+  }
+
+  const getUserFeed = async () => {
+    const userFeed = await userStorage.getAllFeed()
```

Comment:
use getfeedpage(numberofrecords) you want.
it can be a fixed number or you can decide on time span and keep calling getfeedpage until record is older than timespan

Reply:
I try with this but not returned the expected result, this returns an empty array. I tested with a fixed value
For example:
`const userFeed = await userStorage.getFeedPage(5)`

Reply:
this is what we use everywhere, getAllFeed is deprecated, you are doing something wrong